spec/features/user_votes_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+feature "User Votes ", %Q{
+  As a registered user
+  I want to vote on reviews
+  so that I can tell people which reviews are more helpful
+
+  [ ] User can vote up or down
+  [ ] If user already voted on review, user can change vote
+  [ ] If user already voted on review, user can delete vote
+  } do
+    before(:each) do
+      user = FactoryGirl.create(:user)
+      review = FactoryGirl.create(:review)
+
+      visit new_user_session_path
+
+      fill_in "Email", with: user.email
+      fill_in "Password", with: user.password
+
+      click_button "Log in"
+
+      visit station_path(review.station)
+    end
+
+    scenario 'User votes on review' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.